src/providers/authenticationProvider.ts

```diff
@@ -215,6 +214,111 @@ export class TerraformCloudAuthenticationProvider implements vscode.Authenticati
     this._onDidChangeSessions.fire({ added: added, removed: removed, changed: changed });
   }
 
+  private async promptForToken(): Promise<string | undefined> {
+    const choice = await vscode.window.showQuickPick(
+      [
+        {
+          label: 'Stored User Token',
+          detail: 'Use a token stored in the Terraform CLI configuration file',
+        },
+        {
+          label: 'Existing User Token',
+          detail: 'Enter a token manually',
+        },
+        {
+          label: 'Open to generate a User token',
+          detail: 'Open the Terraform Cloud website to generate a new token',
+        },
+      ],
+      {
+        canPickMany: false,
+        ignoreFocusOut: true,
+        placeHolder: 'Choose a method to enter a Terraform Cloud User Token',
+        title: 'HashiCorp Terraform Cloud Authentication',
+      },
+    );
+    if (choice === undefined) {
+      return undefined;
+    }
+
+    // TODO: Change to production URL
+    const terraformCloudURL = 'https://app.staging.terraform.io/app/settings/tokens?source=terraform-login';
```

Comment:
```suggestion
    const terraformCloudURL = `https://${TerraformCloudHost}/app/settings/tokens?source=terraform-login`;
```

Reply:
I think we should consider using a different `source` here, like `terraform-vscode` or anything other than the existing `terraform-login`. I know it means the modal doesn't come up, but it would make hits from the extension indistinguishable from hits from the CLI if we kept it as is.